addons/l10n_cl/data/l10n_latam.document.type.csv

```diff
@@ -1,52 +1,52 @@
-dc_nc_f_dte,130,61,Nota de Crédito Electrónica,NOTA DE CREDITO,credit_note,N/C,base.cl,True
-dc_s_f_dtn,140,108,SRF Solicitud de Registro de Factura,SOL REGISTRO,,REG,base.cl,False
-dc_ftf_f_dtn,150,901,Factura de ventas a empresas del territorio preferencial ( Res. Ex. N° 1057,FACTURA,invoice,FAC,base.cl,False
-dc_din_f_dtn,160,914,Declaración de Ingreso (DIN),DEC ING,invoice_in,DIN,base.cl,False
-dc_dizf_f_dtn,170,911,Declaración de Ingreso a Zona Franca Primaria,DEC ING,invoice_in,DIN,base.cl,False
-dc_ftt_f_dtn,180,904,Factura de Traspaso,FACT TR,invoice,FTR,base.cl,False
-dc_frr_f_dtn,190,905,Factura de Reexpedición,FACT RX,invoice,FRX,base.cl,False
-dc_bzf_f_dtn,200,906,Boletas Venta Módulos ZF (todas),BOLETA ZF,invoice,BZF,base.cl,False
-dc_fzf_f_dtn,210,907,Facturas Venta Módulo ZF (todas),FACTURA ZF,invoice,FZF,base.cl,False
-dc_tca_f_dtn,220,500,Ajuste aumento Tipo de Cambio (código 500),TC-A,,TCA,base.cl,False
-dc_tcd_f_dtn,230,501,Ajuste disminución Tipo de Cambio (código 501),TC-D,,TCD,base.cl,False
-dc_b_f_dtn,240,35,Boleta de Venta,BOL,invoice,BOL,base.cl,False
-dc_b_e_dtn,250,38,Boleta exenta,BEX,invoice,BEX,base.cl,False
-dc_b_f_dte,260,39,Boleta Electrónica,BEL,invoice,BEL,base.cl,True
-dc_b_e_dte,270,41,Boleta Exenta Electrónica,BXE,invoice,BXE,base.cl,False
-dc_m_f_dtn,280,70,Boleta de Honorarios,BHO,invoice,BHO,base.cl,False
-dc_m_d_dtn,290,71,Boleta de Honorarios Electrónica,BHE,invoice,BHE,base.cl,True
-dc_oc,300,801,Orden de Compra,OC,,OC,base.cl,False
-dc_hes,310,HES,Hoja de Entrada de Servicio (HES),HES,,HES,base.cl,False
-dc_hem,320,HEM,Hoja de Entrada de Materiales (HEM),HEM,,HEM,base.cl,False
-dc_migo,330,MIG,Movimiento de Mercancías (MIGO),MIGO,,MIGO,base.cl,False
-dc_li,340,103,Liquidación,LIQ,,LIQ,base.cl,False
-dc_fe_dte,370,110,Factura de Exportación Electrónica,FCXE,invoice,FCXE,base.cl,False
-dc_ndex_dte,380,111,Nota de Débito de Exportación Electrónica,NDXE,debit_note,NDXE,base.cl,False
-dc_ncex_dte,390,112,Nota de Crédito de Exportación Electrónica,NCXE,credit_note,NCXE,base.cl,False
-dc_ndp,400,802,Nota de pedido,NP,,NP,base.cl,False
-dc_cont,410,803,Contrato,CONT,,CONT,base.cl,False
-dc_resol,420,804,Resolución,RES,,RES,base.cl,False
-dc_prchc,430,805,Proceso ChileCompra,PCHC,,PCHC,base.cl,False
-dc_fichc,440,806,Ficha ChileCompra,FCHC,,FCHC,base.cl,False
-dc_dus,450,807,DUS,DUS,,DUS,base.cl,False
-dc_bl_cemb,460,808,B/L (Conocimiento de embarque),B/L,,B/L,base.cl,False
-dc_awb,470,809,AWB Airway Bill,AWB,,AWB,base.cl,False
-dc_mic_dta,480,810,MIC/DTA,MDT,,MDT,base.cl,False
-dc_cpor,490,811,Carta de Porte,CPR,,CPR,base.cl,False
-dc_res_sna,500,812,Resolución del SNA donde califica Servicios de Exportación,RSN,,RSN,base.cl,False
-dc_pasap,510,813,Pasaporte,PSP,,PSP,base.cl,False
-dc_cd_bol,520,814,Certificado de Depósito Bolsa Prod. Chile.,CRTD,,CRTD,base.cl,False
-dc_vp_pren,530,815,Vale de Prenda Bolsa Prod. Chile,VLPR,,VLPR,base.cl,False
```

Comment:
I am a bit confused by the change of the sequence here.  Why not put the file in the right order and do with the sequence as before?

Reply:
It is important to us, by definition, to maintain the new sequence just for the enabled documents. By now it is irrelevant to hold a sequence for the rest of the document (at least by now). 
We made only a change in the file, which is now ordered by sequence just for visibility purpose, but yet, preserving the new sequence. This has been made at 8bb92fc4e2a

Reply:
ok